### PR TITLE
Fix for cmake

### PR DESCRIPTION
**Pull request by kuznetsovmoci**

### CODE REVIEW

File: opendds_cmake_module/CMakeLists.txt [hunk 1]
```diff
@@ -14,7 +14,4 @@ if(BUILD_TESTING)
 endif()
 
 # Make this package available to other ROS-2 packages.
-ament_package()
-
-# Install CMake utilities.
-install(PROGRAMS ${PROJECT_NAME}/${PROJECT_NAME} DESTINATION lib/${PROJECT_NAME})
\ No newline at end of file
+ament_package()
```
Comment (iguessthislldo) [inline opendds_cmake_module/CMakeLists.txt]: This is missing a final newline

Comment (kuznetsovmoci) [reply opendds_cmake_module/CMakeLists.txt]: Added

File: opendds_cmake_module/CMakeLists.txt [hunk 2]
```diff
@@ -14,7 +14,4 @@ if(BUILD_TESTING)
 endif()
 
 # Make this package available to other ROS-2 packages.
-ament_package()
-
-# Install CMake utilities.
-install(PROGRAMS ${PROJECT_NAME}/${PROJECT_NAME} DESTINATION lib/${PROJECT_NAME})
```
Comment (adamsj-ros) [inline opendds_cmake_module/CMakeLists.txt:20]: Why is the install removed? I'm  guessing that's fine but did you check that the rmw_opendds still works?

Comment (kuznetsovmoci) [reply opendds_cmake_module/CMakeLists.txt:20]: Yes - I removed it after I had to remove some other code from rmw_opendds

**Pull request merged**